launcher/net/Download.cpp

```diff
@@ -113,10 +114,15 @@ void Download::executeTask()
     }
 
     request.setHeader(QNetworkRequest::UserAgentHeader, APPLICATION->getUserAgent().toUtf8());
+    // TODO remove duplication and use constant
     if (APPLICATION->capabilities() & Application::SupportsFlame
-            && request.url().host().contains("api.curseforge.com")) {
+            && request.url().host() == "api.curseforge.com") {
```

Comment:
I suppose changing the `QString`s in `BuildConfig` to `QUrl`s incurs too many changes. Even so, you can do something like this for now

```cpp
request.url().host() == QUrl(BuildConfig::FLAME_BASE_URL).host()
```

Reply:
or maybe just duplicate the entries in BuildConfig and add a QUrl version of the strings additionally

Reply:
:P BuildConfig is actually a variable. this should be changed... (PascalCase doesn't feel right here)